robot_skills/package.xml

```diff
@@ -58,12 +58,6 @@
   <build_depend>cb_planner_msgs_srvs</build_depend>
   <run_depend>cb_planner_msgs_srvs</run_depend>
 
-  <build_depend>octomap_msgs</build_depend>
-  <run_depend>octomap_msgs</run_depend>
-
-  <build_depend>cb_planner_msgs_srvs</build_depend>
-  <run_depend>cb_planner_msgs_srvs</run_depend>
-
```

Comment:
You are doing too much stuff changes in one PR. It makes it really hard to review. Can you please make PR's based on logical chunks of work?

Reply:
You are right. Do you want me to create separate PR's now? Or only for future PR's

Reply:
For now, id' revert the removal of this dep and make an issue/PR of cleanup of the dependencies. Which has low prio.